qa/L0_trt_shape_tensors/test.sh

```diff
@@ -45,7 +45,7 @@ CLIENT_LOG="./client.log"
 SHAPE_TENSOR_TEST=trt_shape_tensor_test.py
 
 SERVER=/opt/tritonserver/bin/tritonserver
-SERVER_ARGS="--model-repository=`pwd`/models"
+SERVER_ARGS="--model-repository=`pwd`/models --log-verbose=1"
```

Comment:
Need this change?

Reply:
I added this to enable more detailed logging, which will be useful for analysis in case of job failure. Since verbose logging was not currently enabled, we had to reproduce the issue again locally to know the behavior of server.